### PR TITLE
Docs edits

### PR DESCRIPTION
**Pull request by discdiver**
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [ ] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ tags:
 
 Prefect 2 is licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0), an [OSI approved](https://opensource.org/licenses/Apache-2.0) open-source license. If you have any questions about licensing, please [contact us](mailto:hello@prefect.io).
 
+### Is the Prefect v2 Cloud URL different than the Prefect v1 Cloud URL?
```
Comment (tpdorsey) [inline docs/faq.md:19]:
```suggestion
### Is the Prefect 2 Cloud URL different than the Prefect 1 Cloud URL?  
```

File: docs/faq.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ tags:
 
 Prefect 2 is licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0), an [OSI approved](https://opensource.org/licenses/Apache-2.0) open-source license. If you have any questions about licensing, please [contact us](mailto:hello@prefect.io).
 
+### Is the Prefect v2 Cloud URL different than the Prefect v1 Cloud URL?  
+
+Yes. Prefect Cloud for v2 is at [app.prefect.cloud/](https://app.prefect.cloud) while Prefect Cloud for v1 is at [cloud.prefect.io](https://cloud.prefect.io/).
```
Comment (tpdorsey) [inline docs/faq.md:21]:
```suggestion
Yes. 

Prefect Cloud 2 is at [https://app.prefect.cloud/](https://app.prefect.cloud).

Prefect Cloud 1 is at [https://cloud.prefect.io](https://cloud.prefect.io/).
```

File: docs/faq.md [hunk 3]
```diff
@@ -77,7 +81,7 @@ Note that when tasks are called on constant values, they cannot detect their ups
 
 ### Can I enforce ordering between tasks that don't share data?
 
-Yes! For more information, see the [`Tasks` section][/concepts/tasks/#wait-for].
+Yes! For more information, see the [`Tasks` section](/concepts/tasks/#wait-for).
```
Comment (tpdorsey) [inline docs/faq.md:84]:
```suggestion
Yes! For more information, see the [Tasks section](/concepts/tasks/#wait-for).
```

Comment (tpdorsey) [reply docs/faq.md:84]: Not sure why that was styled as code.

File: docs/faq.md [hunk 4]
```diff
@@ -101,20 +105,22 @@ See the [Installation](/getting-started/installation/) documentation and [Window
...
 
-To use PostgreSQL, users must provide the [connection string][prefect.settings.PREFECT_ORION_DATABASE_CONNECTION_URL] for a running database via the `PREFECT_ORION_DATABASE_CONNECTION_URL` environment variable.
+Prefect Cloud users do not need to worry about the Prefect database. Prefect Cloud uses PostgreSQL on GCP behind the scenes. To use PostgreSQL with a self-hosted Orion server, users must provide the [connection string][prefect.settings.PREFECT_ORION_DATABASE_CONNECTION_URL] for a running database via the `PREFECT_ORION_DATABASE_CONNECTION_URL` environment variable.
 
 ### What databases does Prefect support?
 
-Prefect works with SQLite and PostgreSQL. New Prefect installs default to a SQLite database hosted at `~/.prefect/orion.db`.
+A self-hosted Prefect Orion server can work with SQLite and PostgreSQL. New Prefect installs default to a SQLite database hosted at `~/.prefect/orion.db` on Mac or Linux machines. SQLite and PostgreSQL are not installed by Prefect.
```
Comment (tpdorsey) [inline docs/faq.md:112]:
```suggestion
A self-hosted Prefect Orion server can work with SQLite or PostgreSQL. New Prefect installs default to a SQLite database hosted at `~/.prefect/orion.db` on Mac or Linux machines. SQLite and PostgreSQL are not installed by Prefect.
```

File: docs/index.md [hunk 5]
```diff
@@ -100,6 +104,10 @@ Prefect makes it easy to send tasks to remote clusters for distributed parallel
 
 Prefect is often used with [Docker and Kubernetes](/concepts/deployments/). 
 
+**Automations**
+
+Configure all sorts of actions to run in response to triggers via [automations](./io/ui/automations/).
```
Comment (tpdorsey) [inline docs/index.md:109]: That page URL doesn't look right. I think it's just /ui/automations/.

**Review by tpdorsey — APPROVED** [review]
Minor suggested edits. Looks good.

File: docs/index.md (hunk 5 above)
Comment (discdiver) [inline docs/index.md:109]:
```suggestion
Configure all sorts of actions to run in response to triggers via [automations](ui/automations/).
```

Comment (discdiver) [reply docs/index.md:109]: good catch!

**Review by discdiver — COMMENTED** [review]
Good suggestions @tpdorsey!

**Pull request merged**